CodeEdit/Features/NavigatorArea/SourceControlNavigator/History/Views/CommitListItemView.swift

```diff
@@ -6,10 +6,11 @@
 //
 
 import SwiftUI
+import SwiftGitX
 
 struct CommitListItemView: View {
 
-    var commit: GitCommit
+    var commit: Commit
```

Comment:
Lots of changes here which is fine because of the difference in structure between the two structs. But I'm curious if the other models are also different in `SwiftGitX`?

Makes me wonder if makes sense to actually just map `SwiftGitX` to the models that current exist?